Testing/VelaptorTests/Helpers/AssertExtensions.cs

```diff
@@ -284,21 +282,49 @@ public static void EqualWithMessage(int expected, int actual, string message)
             }
         }
 
-        public static void EqualWithMessage<T>(T expected, T actual, string message)
-            where T : IEquatable<T>
+        /// <summary>
+        /// Verifies that the two enums are equivalent.
+        /// </summary>
+        /// <param name="expected">The expected enum.</param>
+        /// <param name="actual">The actual enum.</param>
+        /// <param name="message">The message to be shown about the failed assertion.</param>
+        public static void EqualWithMessage(Enum expected, Enum actual, string message)
         {
             var assertException = new AssertActualExpectedException(expected.ToString(), actual.ToString(), message);
 
             try
             {
                 Assert.True(expected.Equals(actual), string.IsNullOrEmpty(message) ? string.Empty : message);
```

Comment:
Refactor name of all helper methods with the name `Equal` to `Equals`

Reply:
Upon looking into this, it should be left as a singular word of `Equal` to match the naming of _**XUnit**_